File: pbset6/mario/mario.py
from cs50 import get_int

n = -5

while n < 0 or n > 8:
    n = get_int("Height: ")

for a in range(n):
    for b in range(1, n-a):
        print(" ", end="")
    for c in range(a + 1):
        print("#", end="")
    print("  ", end="")
    for d in range(a + 1):
        print("#", end="")
    print("")

File: pbset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("height ");
    }

    while (n<1 || n>8);
    
    
    
    

    for (int a = 0; a < n; a++)
    {
        for(int b = 1; b < n - a; b++)
        {
            printf(" ");
        }
        for (int b = 0; b <= a; b++)
        {
            printf("#");
        }
        
        printf("  ");
        
        for (int c = 0; c <= a; c++)
        {
            printf("#");
        }
        
        printf("\n");
    }
   
   



}